internal/store/image.go

```diff
@@ -8,42 +8,42 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/docker/app/internal/relocated"
+	"github.com/docker/app/internal/image"
 	"github.com/docker/distribution/reference"
 	multierror "github.com/hashicorp/go-multierror"
 	digest "github.com/opencontainers/go-digest"
 	"github.com/pkg/errors"
 )
 
 //
-type BundleStore interface {
+type ImageStore interface {
```

Comment:
super nit: Is it an `ImageStore` or an `AppImageStore` ?

Reply:
I was also wondering. wanted to avoid too long names (hey, this is Golang :P) but AppImageStore would better describe